js/components/videoQualityPanel.js

```diff
@@ -62,8 +62,16 @@ var VideoQualityPanel = React.createClass({
         this.addAutoButton(bitrateButtons);
       }
       else {
-        var label = Math.round(availableBitrates[i].bitrate/1000) + ' kbps';
-        bitrateButtons.push(<li key={i}><a className={qualityBtn} key={i} onClick={this.handleVideoQualityClick.bind(this, availableBitrates[i].id)}>{label}</a></li>);
+        if (typeof availableBitrates[i].bitrate === "number")
```

Comment:
* add this conditional logic to the first statement so you don't duplicate line 68